src/components/accessories/patientExams/PatientExams.tsx

```diff
@@ -65,6 +66,7 @@ const PatientExams: FC = () => {
       dispatch(deleteLabReset());
       setShouldResetForm(true);
       setShouldUpdateTable(true);
+      dispatch(getLabsByPatientId(patientData?.code));
```

Comment:
I think that this is useless now isnt' it?

Reply:
You're right !